src/otx/algo/detection/heads/base_head.py

```diff
@@ -544,6 +546,9 @@ def export_by_feat(
                 - bboxes (Tensor): Has a shape (num_instances, 4),
                   the last dimension 4 arrange as (x1, y1, x2, y2).
         """
+        if batch_img_metas is None:
```

Comment:
I think if `batch_img_metas` is None then error is raised at line 573

Reply:
Done! 
2e709ad